ci-operator/config/maistra/prometheus/maistra-prometheus-maistra-2.1.yaml

```diff
@@ -2,7 +2,7 @@ build_root:
   image_stream_tag:
     name: maistra-builder
     namespace: ci
-    tag: "2.1"
+    tag: "2.3"
```

Comment:
The file name sould be ci-operator/config/maistra/prometheus/maistra-prometheus-maistra-2.**3**.yaml?

Reply:
no